test/built-ins/Temporal/Duration/prototype/total/relativeto-plaindatetime.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2024 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.duration.prototype.total
+description: >
+  relativeTo is a Temporal.PlainDateTime object.
+info: |
+  Temporal.Duration.prototype.total ( totalOf )
+
+  ...
+  7. Let relativeToRecord be ? GetTemporalRelativeToOption(totalOf).
+  ...
+
+  GetTemporalRelativeToOption ( options )
+
+  1. Let value be ? Get(options, "relativeTo").
+  ...
+  5. If value is an Object, then
+    ...
+    c. If value has an [[InitializedTemporalDateTime]] internal slot, then
+      i. Let plainDate be ! CreateTemporalDate(value.[[ISODateTime]].[[ISODate]], value.[[Calendar]]).
+      ii. Return the Record { [[PlainRelativeTo]]: plainDate, [[ZonedRelativeTo]]: undefined }.
+    ...
+features: [Temporal]
+---*/
+
+var duration = new Temporal.Duration(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
+
+var relativeToDate = new Temporal.PlainDateTime(1970, 1, 1);
```

Comment:
No, it's intentionally a `PlainDateTime`. `PlainDate` inputs are already covered (GetTemporalRelativeToOption, step 5.b), but there wasn't any coverage for `PlainDateTime` (GetTemporalRelativeToOption, step 5.c.)

Reply:
Are you really sure? The line below it would still provide coverage for `PlainDateTime`. If this line is supposed to be `PlainDateTime` as well, then it looks to me like the test is executing the exact same thing twice and asserting that the results are equal. So maybe there's something else missing?

Reply:
Oh, you're right. GH was just showing not enough line context, so I didn't see the other variable.